data/locale/jack_mixer-fr.po

```diff
@@ -0,0 +1,677 @@
+msgid "<b>Quit application?</b>"
+msgstr "<b>Quitter l'application ?</b>"
+
+#: jack_mixer/app.py:678 jack_mixer/app.py:681 jack_mixer/app.py:682
+msgid ""
+"All jack_mixer ports will be closed and connections lost,\n"
+"stopping all sound going through jack_mixer.\n"
+"\n"
+"Are you sure?"
+msgstr ""
+"Tous les ports de jack_mixer seront fermés et les connexions perdues,\n"
+"ce qui arrêtera tout son passant par jack_mixer.\n"
+"\n"
+"Êtes-vous sûr ?"
+
+# Don't translate this unless you want default channel names to be localized
+#: jack_mixer/app.py:743 jack_mixer/app.py:746 jack_mixer/app.py:747
+msgid "Input"
+msgstr "Entrée"
+
+# Don't translate this unless you want default channel names to be localized
+#: jack_mixer/app.py:746 jack_mixer/app.py:749 jack_mixer/app.py:750
+msgid "Output"
+msgstr "Sortie"
+
+#: jack_mixer/app.py:874 jack_mixer/app.py:877 jack_mixer/app.py:878
+msgid "Are you sure you want to clear all channels?"
+msgstr "Êtes-vous sûr de vouloir nettoyer tous les canaux ?"
+
+#: jack_mixer/app.py:1103 jack_mixer/app.py:1109 jack_mixer/app.py:1110
+msgid "load mixer project configuration from FILE"
+msgstr "charger la configuration du projet de mixeur à partir du FICHIER"
+
+#: jack_mixer/app.py:1110 jack_mixer/app.py:1116 jack_mixer/app.py:1117
+msgid "enable debug logging messages"
+msgstr "activer les messages de journalisation de débogage"
+
+#: jack_mixer/app.py:1128 jack_mixer/app.py:1134 jack_mixer/app.py:1135
+msgid ""
+"Mixer creation failed:\n"
+"\n"
+"{}"
+msgstr ""
+"Impossible de créer le mixeur :\n"
+"\n"
+"{}"
+
+# Don't translate this unless you want JACK output port names to be localized
+#: jack_mixer/channel.py:90
+#, python-brace-format
+msgid "{channel_name} Out"
+msgstr "{channel_name} Sortie"
+
+#: jack_mixer/channel.py:113 jack_mixer/channel.py:1329
+msgid "M"
+msgstr ""
+
+#: jack_mixer/channel.py:122
+msgid "MON"
+msgstr ""
+
+#: jack_mixer/channel.py:569
+msgid "S"
+msgstr ""
+
+#: jack_mixer/channel.py:580
+msgid "Cannot create a channel"
+msgstr "Impossible de créer un canal"
+
+#: jack_mixer/channel.py:795
+msgid "Cannot create an output channel"
+msgstr "Impossible de créer un canal de sortie"
+
+#: jack_mixer/channel.py:940
+#, python-brace-format
+msgid "Channel '{name}' Properties"
+msgstr "Propriétés du canal '{name}'"
+
+#: jack_mixer/channel.py:981
+msgid "Properties"
+msgstr "Propriétés"
+
+#: jack_mixer/channel.py:986
+msgid "_Name"
+msgstr "_Nom"
+
+#: jack_mixer/channel.py:995
+msgid "Mode"
+msgstr "Mode"
+
+#: jack_mixer/channel.py:996
+msgid "_Mono"
+msgstr "_Mono"
+
+#: jack_mixer/channel.py:997
+msgid "_Stereo"
+msgstr "_Stéréo"
+
+#: jack_mixer/channel.py:1002
+msgid "MIDI Control Changes"
+msgstr "Control Changes MIDI"
+
+#: jack_mixer/channel.py:1008
+#, python-brace-format
+msgid ""
+"{param} MIDI Control Change number (0-127, set to -1 to assign next free CC "
+"#)"
+msgstr ""
+"{param} Numéro de Control Change MIDI (0-127, mettre à -1 pour assigner le "
```

Comment:
Are you sure `{param}` is in the right place? Shouldn't this be maybe "Numéro de Control Change MIDI pour {param}"? Or at least "numéro" should be lowercase?

I may be completely wrong, I don't know much about French grammar, unfortunately.

Reply:
Good eyes. Fixed now. Thanks.